models/Github/GithubManager.js

```diff
@@ -149,6 +149,9 @@ class GithubManager {
       else if(channel.id == config.discord_channel_important_admin && securearray.includes("[admin]")) {
         channel.sendEmbed(embed);
       }
+      else if(channel.id == config.discord_channel_maintainer_chat && securearray.includes("[s]")) {
```

Comment:
Will this catch PR's with `[S]`?

See line https://github.com/yogstation13/yogbot/pull/150/files#diff-8b92db89a250cb72db01087b14f45a61R138

Reply:
should do now.